test/unit/winston/transports/file-create-dir.test.js

```diff
@@ -3,15 +3,15 @@
 const fs = require('fs');
 const assert = require('assert');
 const path = require('path');
-const winston = require('../../lib/winston');
+const winston = require('../../../../lib/winston');
 
 /* eslint-disable no-sync */
 
 describe('winston/transports/file/createLogDir', function () {
-  const logDir = path.resolve(__dirname, '../fixtures/temp_logs');
+  const logDir = path.resolve(__dirname, '../../../fixtures/temp_logs');
 
   beforeEach(function () {
-    fs.rmdirSync(logDir);
+    fs.rmSync(logDir, {recursive: true, force: true})
```

Comment:
This was updated to force deletion of this directory. Without this implementation first run of the test causes errors to throw as the `temp_logs` dir does not exist. On the flip side, errors would throw if the directory did exist as it was almost never empty. This is a bit heavy-handed but eliminates the need of more complex error handling when leveraging `rmdirSync()`.

Reply:
This has to be updated to leverage `rimraf` as for some reason in Node 14 I was getting really weird behavior for the second test during the `createWriteStream` operation. For some reason it wasn't creating the directory appropriately. My presumption is that it was an asynchronous issue but I'm not positive.